### PR TITLE
Adding Travis.CI

### PR DESCRIPTION
**Pull request by mcverry**
(also linted with flake8)

### CODE REVIEW

File: mbq/client/__init__.py [hunk 1]
```diff
@@ -1,5 +1,4 @@
-from client import ServiceClient
-from authenticator import Authenticator
-from token_manager import TokenManager
-from storage import DjangoCacheStorage
-
+from client import ServiceClient  # noqa
+from authenticator import Authenticator  # noqa
+from token_manager import TokenManager  # noqa
+from storage import DjangoCacheStorage  # noqa
```
Comment (dursk) [inline mbq/client/__init__.py]: Can we alphabetize this pweeeeeeaaaseeee 😸

File: setup.cfg [hunk 2]
```diff
@@ -0,0 +1,2 @@
+[flake8]
+max-line-length=120
```
Comment (mattbriancon) [inline setup.cfg]: We use 100

Comment (dursk) [inline setup.cfg]: I thought we were standardizing on 100?

**Comment by dursk** [comment]
🏄

File: setup.py [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+with codecs.open('README.md', 'r', 'utf-8') as f:
+    readme = f.read()
+
+setuptools.setup(
+    name='mbq.tokens',
+    long_description=readme,
+    version=__version__,
+    license='Apache 2.0',
+    url='https://github.com/managedbyq/mbq.tokens',
```
Comment (mattbriancon) [inline setup.py]: `s/tokens/client/`

File: setup.py [hunk 4]
```diff
@@ -0,0 +1,34 @@
+import codecs
+import setuptools
+
+__version__ = '0.0.1'
+
+with codecs.open('README.md', 'r', 'utf-8') as f:
+    readme = f.read()
+
+setuptools.setup(
+    name='mbq.tokens',
```
Comment (mattbriancon) [inline setup.py]: `s/tokens/client/`

**Pull request merged**